examples/snapshot/package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.0.0",
   "name": "example-snapshot",
   "dependencies": {
-    "react": "*"
+    "react": "17.0.2"
   },
   "devDependencies": {
-    "@babel/core": "*",
-    "@babel/preset-env": "*",
-    "@babel/preset-react": "*",
-    "babel-jest": "*",
-    "jest": "*",
-    "react-test-renderer": "*"
+    "@babel/core": "^7.11.6",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.12.1",
+    "babel-jest": "^28.0.0-alpha.8",
```

Comment:
These won't be bumped automatically, can we keep them at `*`? Potentially `workspace:*`, but that will mess with people doing copy-paste

Reply:
fixed, please have another look.